src/05-derivatives/01-introduction.md

```diff
@@ -0,0 +1,296 @@
+corresponding subject directory; and so on.
+
+## Derived dataset and pipeline description
+
+As for any BIDS dataset a `dataset_description.json` file MUST be found at the
+top level of the particular pipeline:
+`<dataset>/derivatives/<pipeline_name>/dataset_description.json`
+
+In addition to the keys for raw BIDS datasets,
+derived BIDS datasets include the following REQUIRED, RECOMMENDED or OPTIONAL
+`dataset_description.json` keys
+(a dot in the Key name denotes a key in a subdictionary):
+
+| **Key name**                  | **Description**                                                                                                                                                                                                                              |
+| ----------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| PipelineDescription.Name      | REQUIRED. Name of the pipeline that generated the outputs. In case the derived dataset is stored as a subfolder of the raw dataset this field MUST be a substring of the derived dataset folder name (a.k.a. `<pipeline_name>` - see above). |
+| PipelineDescription.Version   | RECOMMENDED. Version of the pipeline.                                                                                                                                                                                                        |
+| PipelineDescription.CodeURL   | OPTIONAL. URL where the code for the analysis can be found.                                                                                                                                                                                  |
+| PipelineDescription.Container | OPTIONAL. Object specifying the location and relevant attributes of software container image used to produce the derivative. Valid fields in this object include `Type`, `Tag` and `URI`.                                                    |
+| SourceDatasets                | RECOMMENDED. A list of objects specifying the locations and relevant attributes of all source datasets. Valid fields in each object include `URL`, `DOI`, and `Version`.                                                                     |
+
+Example:
+
+```JSON
+{
+    "Name": "FMRIPREP Outputs",
+    "BIDSVersion": "1.4.0",
+    "PipelineDescription": {
+        "Name": "FMRIPREP",
+        "Version": "1.2.5",
+        "Container": {
+            "Type": "docker",
+            "Tag": "poldracklab/fmriprep:1.2.5"
+            }
+        },
+    "SourceDatasets": [
+        {
+            "DOI": "10.18112/openneuro.ds000114.v1.0.1",
+            "URL": "https://openneuro.org/datasets/ds000114/versions/1.0.1",
+            "Version": "1.0.1"
+        }
+    ]
+}
+```
```

Comment:
How soon upon merge of this PR the release would happen?
Ideally I would hope to have #440 finished (which would expand the format of this record), and see how it could be maximally aligned with #439 .

Reply:
The current goal is to have 1 week comment (ending next Monday, June 1), 1 week freeze for any final tasks that need doing (unless something critical is found), and release on June 8.

Is there anything that conflicts with #440 or #439? If so, let's try to address it ASAP. I'll try to have a look this afternoon, but feel free to raise any issues here.